### PR TITLE
Revised conda environment exercise

### PR DESCRIPTION
**Pull request by johanhellsvik**
Revised the conda environment exercise following the discussion on Zulip.

### CODE REVIEW

File: _episodes/03-dependencies.md [hunk 1]
```diff
@@ -195,42 +195,26 @@ To get an idea of what's needed, let's have a look at the
...
-> $ conda activate wordcount
-> ```
-> We now have (roughly) the same environment as specified by the
-> developers of the word-count project. But let's say we want to
-> extend this environment, and share it with colleagues:
 > - Inspect your available environments with `conda info -e`.
-> - Install the `pandas` package using `conda install`.
-> - Add `pandas` with the version you installed to the `requirements.txt` file.
-> - Export the full environment using `conda env export > full_env.yml`, and 
+> - Activate the base directory with `conda deactivate`.
```
Comment (bast) [inline _episodes/03-dependencies.md:201]: this first step is a bit surprising (activate vs deactivate)

Comment (johanhellsvik) [reply _episodes/03-dependencies.md:201]: Good spotting it. I agree it can be revised for clarity. I will deal with it tomorrow.

Comment (johanhellsvik) [reply _episodes/03-dependencies.md]: Thank you for pointing out the need for revision. These first steps have been changed.

Comment (bast) [reply _episodes/03-dependencies.md]: Thanks!

**Pull request merged**